.github/workflows/tagging.yml

```diff
@@ -35,7 +35,6 @@ jobs:
       - name: Set up Git configuration
         run: |
           git config user.name "Databricks SDK Release Bot"
-          git config --unset-all http.https://github.com/.extraheader
```

Comment:
This was likely unnecessary. We will see when running this workflow again on main.